src/styles/components/VerticalNav.scss

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+@import "../common/mixins";
+@import "../common/layout";
+@import "../common/colors";
+@import "../common/fonts";
+
+.clr-vertical-nav {
+    %side-nav-item-height {
+        height: $side-nav-item-height;
+        line-height: $side-nav-item-height;
+    }
+    %side-nav-child-item-height {
+        height: $side-nav-child-item-height;
+        line-height: $side-nav-child-item-height;
+    }
+    %side-nav-active-no-subsection{
+        color: $side-nav-active-font-color!important;
+        background-color: $side-nav-active-background!important;
+        font-weight: $font-weight-medium!important;
+    }
+    color: $side-nav-default-font-color!important;
+    background-color: $side-nav-default-background;
+    font-size: $side-nav-font-size;
+    box-shadow: 4px 0 4px $side-nav-border-color;
+    border: 1px solid $side-nav-border-color;
+    width: 17.5rem;
+    padding-top: 0px;
+    .nav-content {
+        padding-top: 0px!important;
+    }
+    &.has-nav-groups {
+        .nav-group {
+            .nav-group-text {
+                &.active {
+                    font-weight: $font-weight-medium!important;
+                }
+            }
+        }
+       .nav-group-children {
+            .nav-link {
+                font-weight: $font-weight-regular;
+                @extend %side-nav-child-item-height;
+                padding-left: 2.2rem;
+                .nav-text {
+                   @extend %side-nav-child-item-height;
+                }
+                &:active {
+                    @include side-nav-active(false, false);
+                }
+                &.active {
+                    @include side-nav-active(false, false);
+                }
```

Comment:
is it possible to merge these into a single class/entity?

Reply:
Done